Update Add-AppxPackage.md (#1639)
MicrosoftDocs/windows-powershell-docs: docset/windows/appx/Add-AppxPackage.md

```

### -ForceUpdateFromAnyVersion

> This parameter is used to Force a specific version of a package to be staged/registered, regardless of if a higher version is already staged/registered.

Reviewer: ```suggestion
This parameter is used to force a specific version of a package to be staged/registered, regardless of whether a higher version is already staged/registered.
```

```

### -LimitToExistingPackages

> This parameter is used not to download missing referenced packages.

Reviewer: ```suggestion
This parameter is used to prevent missing referenced packages to be downloaded.
```

```

### -RelatedPackages

> An optional element that is used to specify the other optional packages that are specified in the main app package. These packages will not be installed as part of the deployment operation.

Reviewer: ```suggestion
This is an optional element that is used to specify the other optional packages that are specified in the main app package. These packages will not be installed as part of the deployment operation.
```

```

### -RetainFilesOnFailure

> In the case of a failed deployment, If this switch is set to $true it does not remove files that have been created on the target machine during the install process.

Reviewer: ```suggestion
In the case of a failed deployment, if this switch is set to $true, files that have been created on the target machine during the installation process are not removed.
```
